charts/kuberpult/values.yaml

```
@@ -52,6 +52,9 @@ git:
   # Disabling this option does not delete the `/commit` directory.
   enableWritingCommitData: false
 
+  # Kuberpult tries to reduce the number of pushes and can bundle concurrent commits into a single push.
```

Please check at startup of the cd-service, that the value is in a sane range.
```suggestion
  # Kuberpult tries to reduce the number of pushes and can bundle concurrent commits into a single push.
  # Must be 1 <= maximumCommitsPerPush <= 100
```

---

Not sure if we want to set a hard upper bound for this value all. Currently the value is essentially infite and we only see issues for the very largest deployments. Good values for the limit depend on the performance of the git provider and I wouldn't want to set a fixed limit there. I will still add a comment and check for lower bound.